### PR TITLE
Add separate function for tile deserialization, fix error message

### PR DESCRIPTION
**Pull request by alpinus4**
Decoupling entity deserialization and tile deserialization logic would be very inefficient.
Removed the unnecesary error, now we don't get pointless error about failing to deserialize entity.
Errors with deserializing are handled by EntityDeserializer.

Moved tile logic to separate function.

### CODE REVIEW

File: scene/world_updater/SceneRecreator.gd [hunk 1]
```diff
@@ -8,14 +8,13 @@ extends Node
 func recreate_scene(scene_json):
 	for entity_json in scene_json["entities"]:
 		var entity = entity_deserializer.deserialize(entity_json)
-		
 		if !entity:
-			print("[ERROR] Could not deserialize Entity")
 			continue
 		
 		# TODO: this could be done at beginning of the loop by checking type key value
 		if !entity is LynxObject:
```
Comment (filip-szweda) [inline scene/world_updater/SceneRecreator.gd:15]: `if !(entity is LynxAction):`

File: scene/world_updater/entity_deserializer/EntityDeserializer.gd [hunk 2]
```diff
@@ -5,54 +5,49 @@ extends Node
...
-		print("[ERROR] Deserializing unknown Entity name: " + attributes.get("name"))
+		print("[ERROR] Deserializing unknown Object name: " + attributes.get("name"))
 		return null
 	
 	var entity_instance = entity.instantiate()
 	entity_instance._populate(attributes)
 	return entity_instance
 
+func deserialize_tile(attributes: Dictionary):
+		tile_setter.set_tile(attributes.name, Vector2i(attributes.position.get("x"), attributes.position.get("y")))
```
Comment (filip-szweda) [inline scene/world_updater/entity_deserializer/EntityDeserializer.gd:34]: One tab, not two

Comment (filip-szweda) [inline scene/world_updater/entity_deserializer/EntityDeserializer.gd:34]: Check if `name`, `position`, `x`, `y` are available

File: scene/world_updater/entity_deserializer/EntityDeserializer.gd [hunk 3]
```diff
@@ -5,54 +5,49 @@ extends Node
...
+		return null # no entity to be instantiated
```
Comment (filip-szweda) [inline scene/world_updater/entity_deserializer/EntityDeserializer.gd:35]: No need for this comment, I think

**Review by filip-szweda — CHANGES_REQUESTED** [review]

File: scene/world_updater/SceneRecreator.gd (hunk 1 above)
Comment (alpinus4) [reply scene/world_updater/SceneRecreator.gd:15]: Not needed, it is scene recreqator, it only operates on entities, not actions.

Comment (filip-szweda) [reply scene/world_updater/SceneRecreator.gd:15]: Sorry, I meant `if !(entity is LynxObject):` xD

**Review by filip-szweda — APPROVED** [review]

**Pull request merged**